src/xcode/ENA/ENA/Source/Scenes/Home/HomeState.swift

```diff
@@ -100,7 +100,13 @@ class HomeState: ENStateHandlerUpdating {
 	}
 
 	var daysSinceInstallation: Int {
-		store.appInstallationDate?.ageInDays ?? 0
+		guard let appInstallationDate = store.appInstallationDate else {
+			return 0
+		}
+		if Calendar.current.isDateInYesterday(appInstallationDate) {
+			return 1
+		}
+		return appInstallationDate.ageInDays ?? 0
```

Comment:
I think the desired effect is to have it change at midnight for every amount of days since installation, not just for the change from "today" to "yesterday". For clarification see also the duplicate linked in the comments of the bug ticket.
```suggestion
		return Calendar.current.startOfDay(for: appInstallationDate).ageInDays ?? 0
```

Reply:
Even better -> Calendar.autoupdatingCurrent

Reply:
Done